esp/templates/program/modules/studentonsite/survey.html

```diff
@@ -17,67 +16,7 @@
         </table>
```

Comment:
When someone fills out just one class's survey, saying "Thanks for filling out the survey!" feels very final to me, as if there aren't other surveys to fill out. Could we say "Thanks for filling out the survey for ___class____! Don't forget to fill out the rest of the survey _here_!" or something if they have more survey to fill out?

Reply:
Hmm...but we don't want them to fill out the rest of the survey until they've taken all of their classes?

Reply:
Yeah thats kinda what I was thinking about with my suggestion below. I guess I'm just worried that even if they fill out each class survey, they won't realize there's a program survey, since the class surveys appear in the survey tab and they would have to press on the survey tab again to ever even notice the program survey. Im not sure what the best solution is. Maybe a message saying "Dont forget to fill out the program survey here at the end of the day!" Either at the bottom of the schedule or when they submit a class survey? Idk